keadm/cmd/keadm/app/cmd/cloud/init.go

```diff
@@ -112,12 +112,13 @@ func Add2ToolsList(toolList map[string]types.ToolsInstaller, flagData map[string
 	if kubeVer == "" {
 		var latestVersion string
 		for i := 0; i < util.RetryTimes; i++ {
```

Comment:
The for loop is useless here, since any error will make return the function.

Reply:
I think this for loop is for retry setting, any error make it return is what we want.